File: Find Your Villain Name.py
def get_villain_name(birthdate): 
    first = [ "The Evil","The Vile","The Cruel", "The Trashy","The Despicable", "The Embarrassing", "The Disreputable","The Atrocious", "The Twirling",  "The Orange","The Terrifying", "The Awkward"]
    last = ["Mustache", "Pickle", "Hood Ornament", "Raisin", "Recycling Bin", "Potato", "Tomato", "House Cat", "Teaspoon", "Laundry Basket"]
    month = birthdate.month
    day = int(str(birthdate.day)[-1])
    if month==11 and day==18:
        return "The Terrifying Teaspoon"
    else:
        return f'{first[month-1]} {last[day]}'

File: Find Your Villain Name/Find Your Villain Name.js
function getVillainName(birthday){
  const m = ["Evil","Vile","Cruel","Trashy","Despicable","Embarrassing","Disreputable","Atrocious","Twirling","Orange","Terrifying","Awkward"];
  const d = ["Mustache","Pickle","Hood Ornament","Raisin","Recycling Bin","Potato","Tomato","House Cat","Teaspoon","Laundry Basket"];
  const day = parseInt(String(birthday.getDate()).slice(-1));
  const month = birthday.getMonth();
  if(month==10 && day==18){
    return "The Terrifying Teaspoon"
  }else{
    return `The ${m[month]} ${d[day]}`
  }
}
